src/test/unittest/ut_signal.c

```diff
@@ -40,12 +40,31 @@
 /*
  * On Windows, Access Violation exception does not raise SIGSEGV signal.
  * The trick is to catch the exception and... call the signal handler.
- *
- * XXX - add support for registering more than one signal/exception handler
  */
 
-static void (*Sa_handler) (int signum);
-static int Signum;
+struct signal_handler {
```

Comment:
Yep, I agree with the sa_mask thing, my bad.

But I think sigaction should be thread-safe ;)